databases/week3/mealsharing_queries.sql

```diff
@@ -0,0 +1,212 @@
+use mealsharing;
+-- Meal
+-- Get all meals
+SELECT
+  title
+from
+  Meal;
+-- Add a new meal
+INSERT INTO
+  Meal(
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+VALUES(
+    'Rød grød med fløde',
+    'lakkert mad alle skal elske',
+    'sundby',
+    '2021-10-25 06:54:16',
+    15,
+    179.00,
+    '2021-10-26 06:54:16'
+  );
+-- Get a meal with any id,
+select
+  *
+from
+  Meal
+WHERE
+  id = 1;
+-- fx 1 Update  a meal with any id,  fx 1.
+UPDATE
+  Meal
+SET
+  title = "Burger"
+WHERE
+  id = 1;
+-- Update any attribute fx the title or multiple attributes
+UPDATE
+  Meal
+SET
+  title = "small pizza",
+  description = "Pizza for babies"
+WHERE
+  id = 1;
+-- Delete a meal with any id,  fx 1
+DELETE FROM
+  Meal
+WHERE
+  id = 3;
+-- RESERVATION
+  -- Get all reservations
+SELECT
+  *
+FROM
+  Reservation;
+-- Add a new reservation
+INSERT INTO
+  Reservation
+VALUES(
+    6,
```

Comment:
It's better so specify which columns you want to insert and not insert the id, since MySQL auto increments it automatically.

Also, running this query is giving me an error